src/page-layout/SharedPage/SharedPage.jsx

```diff
@@ -1,12 +1,41 @@
+import { useGetFolder } from "data-access/useGetFolder";
 import "./SharedPage.css";
+import { FolderInfo } from "ui/FolderInfo";
+import { SearchBar } from "ui/SearchBar";
+import { CardList } from "ui/CardList";
+import { ReadOnlyCard } from "ui/ReadOnlyCard";
+import { getFolders } from "data-access/getFolders";
+import { useEffect, useState } from "react";
+
+export const SharedPage = () => {
+  const [folders, setFolders] = useState();
```

Comment:
P2:
default value 를 []로 해주시고 하단에서 map 돌리실때 옵셔널 체이닝을 빼시면 좋겠네요.
folders가 빈값일때를 판단할 필요가 없어보여서요.

```suggestion
  const [folders, setFolders] = useState();
```